### PR TITLE
Fixes #7227: Fix tests after #7156

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7227

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

File: tree/30_generic_methods/_classes_copy.cf [hunk 1]
```diff
@@ -49,8 +49,6 @@ bundle agent _classes_copy(source_prefix, destination_prefix)
   classes:
       # this line will not ne evaluated by cfengine if destination_prefix contains a variable that do not exist
       # and it will always be evaluated to true otherwise
-      "destination_not_exists" expression => strcmp("${destination_prefix}", "${destination_prefix}");
-      # previous line has been evaluated
-      "destination_should_exists" expression => "any";
```
Comment (ncharles) [inline tree/30_generic_methods/_classes_copy.cf]: oh, we don't need anymore this voodoo part ?
how come ??

Comment (ncharles) [inline tree/30_generic_methods/_classes_copy.cf]: oh, it's not negative. clever

**Pull request merged**